dnf/automatic/main.py

```diff
@@ -217,13 +217,13 @@ def main(args):
 
 
 def upgrade(base, upgrade_type):
-    if upgrade_type == 'default':
+    if upgrade_type == 'security':
```

Comment:
I don't think it breaks anything, it's just code re-ordering, the original if branch is below.